renderapi/client/client_calls.py

```diff
@@ -598,7 +608,13 @@ def pointMatchClient(stack, collection, tile_pairs,
         transforms that you had applied after it. default= None.
     excludeAllTransforms: bool
         alternative to normalizeForMatching which simply removes all transforms from the list.
-        default=False
+        default = False
+    stackChannels: str or None
+        If specified, option to select which channel is used for the stack.
+        default = None
+    stack2Channels: str or None

```

Comment:
what is the use case for doing cross channel point matches?

Reply:
if there isn't a compelling one I would just have this as "channels" like in the other api

Reply:
Use case: "DAPI_1" vs "DAPI_2" vs "DAPI_3".

As for the test, given the test data, failing with different channels seemed like a reasonable test to show that the data are different.

Reply:
that is a good one:) ok